src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0 .star:nth-child(n) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .star:nth-child(n+2) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 .star:nth-child(n+3) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 .star:nth-child(n+4) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 .star:nth-child(n+5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--5 .star:nth-child(n+6) {
+  background-image: url(images/star.svg);
+}
```

Comment:
let's group this, check ou point 2 in the [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)